server/src/socketHandlers/socketHandlers.ts

```diff
@@ -177,19 +177,45 @@ export default function initializeSocketHandler(io: Server) {
     });
 
     socket.on("start_game", ({ lobbyId, members }) => {
-      // Instantiate relevant classes
-      //const gameCards = initializeGameCards(members);
-      //const playerObjects = initializePlayers(members, gameCards);
-      //const dice = new Dice(SixSidedDie);
-      //game = new QwixxLogic(playerObjects, dice);
       lobbiesMap[lobbyId].startGame();
 
-      // Create path data and players' gameboard states to send back to client
-      const initialPlayersState = lobbiesMap[lobbyId].gameLogic?.players;
+      // Create path data and serialized gamelogic object to the front end
+      const gameLogic = lobbiesMap[lobbyId].gameLogic;
+      const serializedGameLogic = gameLogic ? gameLogic.serialize() : null;
```

Comment:
I was thinking that calling the startGame() method could return the initial game state. That way you wouldn't need to do another search through the lobbiesMap. I also think that the possibility of returning a "null" isn't so good and rather an error should be returned if there truly wasn't any gameLogic. 

I also think that the name "serializedGameLogic" is a bit misleading ad we're not really sending over the logic, we're sending over the initial game state.

Reply:
I was thinking more like 

'''
const initialGameState = lobbiesMap[lobbyID].startGame()

if(initialGameState){io.to(lobbyID).emit...}
'''